docs/modules/ROOT/partials/getting-started/con-registry-serdes-protobuf.adoc

```diff
@@ -5,43 +5,52 @@
 
 = Configure Protobuf SerDe with {registry}
 
-{registry} provides Kafka client serializer and deserializer classes for Google Protobuf to make using Protobuf as easy as possible:
+[role="_abstract"]
+{registry} provides the following Kafka client serializer and deserializer classes for Google Protobuf:
 
-* `io.apicurio.registry.utils.serde.ProtobufKafkaSerializer`
-* `io.apicurio.registry.utils.serde.ProtobufKafkaDeserializer`
+* `io.apicurio.registry.serde.protobuf.ProtobufKafkaSerializer`
+* `io.apicurio.registry.serde.protobuf.ProtobufKafkaDeserializer`
 
 .Configure the Protobuf serializer
 
-You can configure the Protobuf serializer class in the following ways:
+You can configure the Protobuf serializer class as follows:
 
-* {registry} location as a URL
-* Artifact ID strategy 
-* Global ID strategy 
+* {registry} URL
+* Artifact resolver strategy 
+* Global ID strategy  
 * Global ID location
 * Global ID handler
 
```

Comment:
there are new config options for protobuf serializer, check `io.apicurio.registry.serde.protobuf.ProtobufKafkaSerializerConfig`.
In particular, the new option is "apicurio.registry.serde.validation-enabled" (same as for json schema serde)